PactConsumerSwift.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name         = "PactConsumerSwift"
+  s.version      = "0.1.2"
```

Comment:
After this gets merged in, this will need to be update with the next version (0.2 or 0.1.3?) and then this cocoapod can be published (`pod trunk push` once the next tag is set).